### PR TITLE
Tracing: pm support

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#344

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ var fs = require('fs');
 var ipcctl = require('./ipcctl');
 var loopback = require('loopback');
 var loopbackBoot = require('loopback-boot');
+var MinkeLite = require('minkelite');
+var minke_config_options = require('./minke_config.json')
```
Comment (sam-github) [inline lib/server.js]: We use `camelCase` for variable names (capitalized for constructors).

File: lib/server.js [hunk 2]
```diff
@@ -40,6 +42,10 @@ function Server(originalCommand, baseDir, listenPort, controlPath) {
   // Set up the /favicon.ico
   this._app.use(loopback.favicon());
 
+ // set up minkelite
+  minke_config_options.port = this._listenPort
```
Comment (sam-github) [inline lib/server.js]: Why does it need the port? It doesn't `listen()`.

**Comment by sam-github** [comment]
Looks OK for POC. Not sure where/if to merge to so it ends up on staging.

Maybe a `v3.x` branch? @kraman ?

Also, does this all work? Have you got it running for @seanbrookes remotely, or does he need it on staging so he can run it locally?

**Comment by Setogit** [comment]
@seanbrookes  has got this stuff working locally for his Arc side of the work.

**Comment by chandadharap** [comment]
@sam-github @seanbrookes is using a local Concurix server and is not waiting on this integration for his Spike. But this is not yet complete.

**Comment by sam-github** [comment]
All is good, thanks, but don't merge until feature is complete.

**Comment by sam-github** [comment]
LGTM

File: lib/service-manager.js [hunk 3]
```diff
@@ -4,8 +4,11 @@ var configDefaults = require('./config').configDefaults;
 var debug = require('debug')('strong-pm:service-manager');
 var util = require('util');
 
-function ServiceManager(server) {
+function ServiceManager(server, enableTracing) {
   this.server = server;
+  this.enableTracing = enableTracing;
+  if (enableTracing)
+    configDefaults.start = 'sl-run --cluster=cpu --trace';
```
Comment (sam-github) [inline lib/service-manager.js]: @kraman @Setogit shouldn't this be something like `configDefaults.start += ' --trace';`, to not assume that --cluster=cpu is current start command?

**Comment by sam-github** [comment]
Option wasn't documented in usage (don't bother adding to README, I'm stripping usage out of readme in #157)

File: lib/service-manager.js (hunk 3 above)
Comment (kraman) [inline lib/service-manager.js]: fixed

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by rmg** [comment]
LGTM.

**Pull request merged**